common-design/sass/cd-header/_cd-nav.scss

```diff
@@ -260,27 +272,27 @@ ul.cd-nav__grandchild-nav {
   }
 
   ul.cd-nav__child-nav {
-    background: $cd-mid-bg-color;
+    background: $cd-white-bg-color;
+    box-shadow: 0 1px 8px rgba($cd-dark-bluey-grey, 0.8);
     left: -1px;
     margin: 0;
     max-width: 300px;
     min-width: 200px;
     padding: 20px 0;
-    top: $cd-site-header-height;
+    top: calc(#{$cd-site-header-height} + 3px); // border-bottom height
```

Comment:
When a variable seems like overkill, another 👍 to documenting where a seemingly-arbitrary number comes from.